### PR TITLE
add seedfile validation

### LINKED ISSUE #3: validate seed file before it's added to ISO
It's a little bit annoying to install your ISO then find out during installation that debian finds an error in your preseed file. Luckily there is a way to validate the file beforehand.

### PR DESCRIPTION
**Pull request by oregonpillow**
Closes #3

### CODE REVIEW

File: mkiso [hunk 1]
```diff
@@ -165,6 +165,15 @@ preseed_arm64() {
   done
 }
 
+# Validate the preseed file
+check_debconf_selections() {
+  local updated_seed_file="${1}"
+  debconf-set-selections --checkonly $updated_seed_file || {
```
Comment (chuckn246) [inline mkiso]: Please quote and wrap this var in curly braces.

**Review by chuckn246 — CHANGES_REQUESTED** [review]

File: mkiso (hunk 1 above)
Comment (chuckn246) [reply mkiso]:
```bash
"${updated_seed_file}"
```

Comment (oregonpillow) [reply mkiso]: done

File: mkiso [hunk 2]
```diff
@@ -168,7 +168,7 @@ preseed_arm64() {
 # Validate the preseed file
 check_debconf_selections() {
   local updated_seed_file="${1}"
-  debconf-set-selections --checkonly $updated_seed_file || {
+  debconf-set-selections --checkonly ${updated_seed_file} || {
```
Comment (chuckn246) [inline mkiso]: Please quote it too

**Review by chuckn246 — CHANGES_REQUESTED** [review]

File: mkiso (hunk 2 above)
Comment (oregonpillow) [reply mkiso]: done

Comment (chuckn246) [reply mkiso]: Thank you, I want to verify this one before approving.

Looking at the [Custom Preseed docs](https://wiki.debian.org/DebianInstaller/Preseed#Custom_preseed_files) you linked in the issue, I want to ensure [debconf-utils](https://packages.debian.org/bookworm/debconf-utils) is available and installed.

Comment (chuckn246) [reply mkiso]: `debconf-set-selections` is available in a couple of my Debian VMs without debconf-utils installed.

**Review by chuckn246 — APPROVED** [review]

**Comment by chuckn246** [comment]
Thank you for your contribution :)

File: mkiso (hunk 2 above)
Comment (oregonpillow) [reply mkiso]: it seems to be available from `debconf-utils` as mentioned [here](https://wiki.debian.org/PackageManagement/Preseed) .

package: https://packages.debian.org/sid/debconf-utils 

should i add it to the requirements in the readme?

**Pull request merged**